google/cloud/bigquery/_pandas_helpers.py

```diff
@@ -283,7 +284,13 @@ def bq_to_arrow_schema(bq_schema):
     return pyarrow.schema(arrow_fields)
 
 
-def default_types_mapper(date_as_object: bool = False):
+def default_types_mapper(
+    date_as_object: bool = False,
+    bool_dtype: Union[Any, None] = pandas.BooleanDtype(),
```

Comment:
Pandas is an optional dependency. Pretty sure we can't do this. Let's use a "sentinel" value as in https://stackoverflow.com/a/60605919/101923 to avoid initializing pandas objects at import time.

Reply:
Good points. Fixed with your suggestions.